app/src/main/res/layout/set_profile_8.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".profile.Profile8">
+
+    <Button
+        android:id="@+id/button_p8"
+        android:layout_width="231dp"
+        android:layout_height="61dp"
+        android:clickable="false"
+        android:onClick="startRecordingActivity"
+        android:text="@string/continue_str"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.955" />
+
+    <TextView
+        android:id="@+id/text1_p8"
+        android:layout_width="230dp"
+        android:layout_height="196dp"
+        android:text="@string/present_yourself_record"
+        android:textColor="#000000"
+        android:textSize="36sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.192"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.167" />
+
+    <ProgressBar
+        android:id="@+id/progressBar_p8"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="413dp"
+        android:layout_height="62dp"
+        android:progress="89"
+        app:layout_constraintBottom_toTopOf="@+id/text_p2"
```

Comment:
This seems to come from a copy-paste of another class, should be removed from constraints.

Reply:
Seems to be fixed! All good.